src/omv/imlib/jpegd.c

```diff
@@ -1,13 +1,25 @@
-/*
- * This file is part of the OpenMV project.
- *
- * Copyright (c) 2013-2021 Ibrahim Abdelkader <iabdalkader@openmv.io>
- * Copyright (c) 2013-2021 Kwabena W. Agyeman <kwagyeman@openmv.io>
- *
- * This work is licensed under the MIT license, see the file LICENSE for details.
- *
- * Baseline JPEG decoder.
- */
+//
+// JPEG Decoder
+//
+// written by Larry Bank
+// bitbank@pobox.com
+// Arduino port started 8/2/2020
+// Original JPEG code written 26+ years ago :)
+// The goal of this code is to decode baseline JPEG images
+// using no more than 18K of RAM (if sent directly to an LCD display)
+//
+// Copyright 2020 BitBank Software, Inc. All Rights Reserved.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//    http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//===========================================================================
+//
```

Comment:
```suggestion
/*
 * This file is part of the OpenMV project.
 * Copyright (c) 1995-2020 Larry Bank bitbank@pobox.com
 * Copyright (c) 2020-2024 Ibrahim Abdelkader <iabdalkader@openmv.io>
 * Copyright (c) 2020-2024 Kwabena W. Agyeman <kwagyeman@openmv.io>
 *
 * This work is licensed under the MIT license, see the file LICENSE for details.
 *
 * Baseline JPEG decoder.
 */
```

We have a bunch of contributions to this file, plus we've paid for contract work to deliver this. I think it's fair if we're mentioned starting when this work was added to the repo and moving forward.